api/src/main/java/com/messagebird/MessageBirdServiceImpl.java

```diff
@@ -191,6 +204,16 @@ public void deleteByID(String request, String id) throws UnauthorizedException,
         HttpURLConnection connection = null;
         InputStream inputStream = null;
 
+        if (METHOD_PATCH.equals(method)) {
```

Comment:
You may want to do an case insensitive check?

Reply:
Resolved in https://github.com/messagebird/java-rest-api/pull/44/commits/0d97d8f3e87eb82fcecdc951c0424bc91061057d